### PR TITLE
2.3.4

### PR DESCRIPTION
**Pull request by oqyh**
Some Cleanup

### CODE REVIEW

**Comment by oqyh** [comment]
Add HalfBuy

**Comment by oqyh** [comment]
Add Bomb Announcement

File: RetakesAllocator/Menus/MenuManager.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ public enum MenuType
     NextRoundVote,
 }
 
-public class MenuManager
+public class Menu_Manager
```
Comment (B3none) [inline RetakesAllocator/Menus/MenuManager.cs:15]: Why is this renamed to `Menu_Manager`, follow the existing project naming convention.

Comment (oqyh) [reply RetakesAllocator/Menus/MenuManager.cs:15]: because of css has same name

https://github.com/roflmuffin/CounterStrikeSharp/blob/71c694b52ea32fe60a0972b9c775abb09fb52e96/managed/CounterStrikeSharp.API/Modules/Menu/MenuManager.cs#L21

Comment (oqyh) [reply RetakesAllocator/Menus/MenuManager.cs:15]: otherwise you will stuck with Obsolete("Use MenuManager.OpenChatMenu instead") cuz you have 2 same names

https://github.com/roflmuffin/CounterStrikeSharp/blob/71c694b52ea32fe60a0972b9c775abb09fb52e96/managed/CounterStrikeSharp.API/Modules/Menu/ChatMenu.cs#L83
